src/scripts/inputHandler.ts

```diff
@@ -0,0 +1,18 @@
+import readline from "readline";
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+export function handleUserInput(): Promise<string> {
+  return new Promise((resolve) => {
+    rl.question(
+      "Please input the token you received in the browser: ",
```

Comment:
It's probably better to move this string as an argument for this function. This way we can make it truly reusable. And `userInput` might be a more appropriate name, since we're just returning the user input, not handling it.

So we would be calling this function like this: 
```
const token = await userInput("Please input the token you received in the browser: ");
```

instead of this:

```
const token = await handleUserInput();
```

Reply:
Good point.